api4/drafts.go

```diff
@@ -121,7 +121,14 @@ func deleteDraft(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	draft, err := c.App.GetDraft(userID, channelID, rootID)
 	if err != nil {
-		c.Err = err
+		switch {
+		case err.StatusCode == http.StatusNotFound:
+			// If the draft doesn't exist in the server, we don't need to delete.
+			mlog.Error("Unable to find the draft", mlog.Err(err))
```

Comment:
0/5 I'm not sure how often this happens, but maybe we don't want to log this at the error level? It could end up spamming the error log if it happens often enough.

Reply:
@ashishbhate Would it be better as a warning or info? It would happen pretty frequently so I agree with not cluttering the error logs.

Reply:
Since we're specifically checking for this error and ignoring it, I don't think we should be logging it.

IIUC, there is going to be some follow-up work for this. In that case, I think maybe log at the DEBUG level so that it won't be outputted by default, but someone debugging can easily find it.

Reply:
Yep, there's gonna be a follow-up here https://mattermost.atlassian.net/browse/MM-48893 to reduce the number of requests (or eliminate them entirely ideally). A `debug` level log makes perfect sense here, totally forgot that was an option as well 😅